### PR TITLE
Closes #62: Add license model

### LINKED ISSUE #62: Add license model

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: src/main/resources/META-INF/persistence.xml [hunk 1]
```diff
@@ -1,13 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <persistence version="1.0" xmlns="http://java.sun.com/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_1_0.xsd">
-    <persistence-unit name="VDCNet-ejbPU" transaction-type="JTA">
+    <persistence-unit name="VDCNet-ejbPU" transaction-type="RESOURCE_LOCAL">
```
Comment (MrK191) [inline src/main/resources/META-INF/persistence.xml]: Is there a reason for changing transaction type?

File: src/main/java/edu/harvard/iq/dataverse/license/License.java [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+            }
+        }
+        return false;
+    }
+    
+    /**
+     * Returns localized version of license name
+     * if it exists or {@link #getName()} otherwise.
+     */
+//    @Transient
```
Comment (MrK191) [inline src/main/java/edu/harvard/iq/dataverse/license/License.java]: Left out?

**Review by MrK191 — CHANGES_REQUESTED** [review]

File: src/main/resources/META-INF/persistence.xml (hunk 1 above)
Comment (madryk) [reply src/main/resources/META-INF/persistence.xml]: Forgot to revert to original version of persistence.xml (this is the only way for my ide to generate sql from jpa)

File: src/main/java/edu/harvard/iq/dataverse/license/License.java (hunk 2 above)
Comment (madryk) [reply src/main/java/edu/harvard/iq/dataverse/license/License.java]: Will remove it - is not needed

**Review by MrK191 — APPROVED** [review]

**Review by MrK191 — APPROVED** [review]

**Pull request merged**